packages/config/config/devices/0x021f/dmms1.json

```diff
@@ -14,6 +14,28 @@
 		"max": "255.255"
 	},
 	"supportsZWavePlus": true,
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Basic_Set",
+			"description": "The Motion Detector sends a BASIC_SET command to directly trigger devices (like a siren, chime, etc.) when motion is detected and when it stops.",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Notification_Report",
+			"description": "Supports up to 5 members and the Motion Detector sends it a NOTIFICATION_REPORT when motion is detected and when it stops.",
```

Comment:
```suggestion
			"description": "Send a Notification Report when motion is detected and when it stops.",
```

Reply:
We try to reduce prose in descriptions and labels to a minimum. Try not to repeat info that is available otherwise (like maximum number of members).